app/models/measure_type.rb

```diff
@@ -48,6 +48,13 @@ def safeguard?
     SAFEGUARD_TYPES.include?(id)
   end
 
+  def abbreviation
+    return 'Prohibition' if description.scan(/Prohibition/i).present?
```

Comment:
Works fine but for future reference, `.match?(/Prohibition/i)` would be quicker 

-> `#scan` will need to scan the entire string for all matches where as `#match?` can just return as soon as it finds the first match